Documentation/Volumes.md

```diff
@@ -0,0 +1,42 @@
+# Attaching volume
+Capstan supports attaching any number of pre-prepared volumes to the instance. Currenly, you need to prepare
+a volume manually i.e. with external tools, and then export it into a file on your host. During the `capstan run`
+you then provide path to this file together with some metadata.
+
+## Creating a new volume
+You can create a new volume using qemu-img tool:
+
+```bash
+$ qemu-img create -f qcow2 ./volume.img 1G
+```
+
+## Attaching volumes
+Tell Capstan where your volume is and it will attach it to the instance on run:
+
+```bash
+$ capstan run demo --volume ./volume.img
```

Comment:
Just for my personal reference: will this get stored in the instance metadata so that on a consecutive run of the same instance I won't need to add the `--volume` part?